### PR TITLE
PowerStore: Volume limits support in Installation Wizard

### PR DESCRIPTION
**Pull request by suryagupta4**
# Description
This PR is for adding volume limits feature support in CSM Installation Wizard for PowerStore.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/878 |

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

# Verification
Volume Limits (Max Volumes Per Node) parameter added in UI
vol-limits-ui

Parameter added in download values.yaml in case of Helm & sample.yaml in case of Operator for PowerStore
helm-values-yaml
operator-sample-yaml

Verified by installing the driver. Volume limit is getting set in the env as expected
env-set

### CODE REVIEW

**Review by niranjan-n1 — DISMISSED** [review]
LGTM

File: content/docs/deployment/csminstallationwizard/src/static/js/ui-functions.js [hunk 1]
```diff
@@ -302,6 +307,9 @@ function displayModules(installationType, driverName, CONSTANTS_PARAM) {
 			$("#authorization").prop('checked', false);
 			$(".storage-capacity").show();
 			$(".resiliency").show();
+			if (document.getElementById("csm-version").value != "1.7.0") {
```
Comment (niranjan-n1) [inline content/docs/deployment/csminstallationwizard/src/static/js/ui-functions.js:310]:
```suggestion
			if (document.getElementById("csm-version").value !== "1.7.0") {
```

**Review by niranjan-n1 — APPROVED** [review]

**Review by samihan-dell — APPROVED** [review]

**Review by AkshaySainiDell — APPROVED** [review]
lgtm

**Review by niranjan-n1 — APPROVED** [review]

**Review by alankar-verma — APPROVED** [review]
lgtm

**Comment by chimanjain** [comment]
Should this be done for PowerScale and PowerMax driver as they also support Volume Limits?

**Comment by suryagupta4** [comment]
> Should this be done for PowerScale and PowerMax driver as they also support Volume Limits?

Yes, if they already have support for this feature. Minimal changes would be required as this PR has most of the changes which are common.

**Comment by chimanjain** [comment]
Yes, it's supported, if it's okay we can modify the template files to make that field dynamic, i.e. field name is `X_CSI_MAX_VOLUMES_PER_NODE`.

**Review by sharont58 — APPROVED** [review]

**Review by chimanjain — APPROVED** [review]

**Pull request merged**